docker/build

```diff
@@ -11,6 +11,11 @@ set -e
 rm -rf docker/rvt/rvt-patch-llvm
 cp -r `pwd`/rvt-patch-llvm docker/rvt/rvt-patch-llvm
 
+# Copy cargo-verify over
+(cd cargo-verify; cargo clean)
```

Comment:
Alternatively (you don't have to change):
```shell
cargo clean --manifest-path cargo-verify/Cargo.toml
```

Reply:
Done